migrations/versions/969126bd800f_.py

```diff
@@ -21,18 +21,21 @@
 
 
 def upgrade():
+    dashboards = prefix_schema("dashboards")
+    widgets = prefix_schema("widgets")
+
     # Update widgets position data:
     column_size = 3
     print("Updating dashboards position data:")
-    dashboard_result = db.session.execute("SELECT id, layout FROM dashboards")
+    dashboard_result = db.session.execute(f"SELECT id, layout FROM {dashboards}")
     for dashboard in dashboard_result:
         print("  Updating dashboard: {}".format(dashboard["id"]))
         layout = simplejson.loads(dashboard["layout"])
 
         print("    Building widgets map:")
         widgets = {}
         widget_result = db.session.execute(
-            "SELECT id, options, width FROM widgets WHERE dashboard_id=:dashboard_id",
+            f"SELECT id, options, width FROM {widgets} WHERE dashboard_id=:dashboard_id",
```

Comment:
hmmm.. we should be able to set the default schema on the connection to be less invasive? also why do we care about historical upgrades. they do a stamp with current afaiks

Reply:
I was being thorough, but it probably isn't strictly needed on the migrations.
As for setting is on the connection, that doesn't work here because these are executing the queries directly, they don't go through a translation map like if it was using the `Table` object to generate the select.

Reply:
@kapilt I could modify the direct execute queries to use the `Table` objects to generate the select statements and then the schema on the connection engine would work.